tests/suite/test_requirements.py

```diff
@@ -71,6 +71,21 @@ def check(network, args, *nargs, **kwargs):
     return ensure_reqs(check)
 
 
+def sufficient_member_count():
```

Comment:
In practice, when we randomise the test suite (i.e. Daily build), the tests marked with this (`test_retire_member`) will often be skipped as the recovery threshold is set to the initial number of members (3) and there is only one test in the suite that adds a new member. 

I'm OK with that for now as we run the test non-randomised in the CI anyway. But if we want to extend this, we could make sure we run the `test_add_member` when this condition fails.